AppInspector/rules/default/components/load_dll.json

```diff
@@ -44,17 +36,13 @@
     "id": "AI005510",
     "description": "Component: Windows DLL",
     "applies_to": [ "rust" ],
-    "tags": [
-      "Component.Executable.Microsoft.DLL"
-    ],
+    "tags":[ "Component.Executable.Microsoft.DLL" ],
```

Comment:
I think some of the patterns (especially libloading) don't just apply to windows. Maybe the tag should be "Component.Executable.SharedLibraryLoading" or something.